jaeger-core/src/main/java/io/jaegertracing/JaegerTracer.java

```diff
@@ -220,23 +223,24 @@ public void close() {
     }
 
     @Override
-    public io.opentracing.Tracer.SpanBuilder asChildOf(io.opentracing.SpanContext parent) {
+    public Tracer.SpanBuilder asChildOf(SpanContext parent) {
```

Comment:
Should all the return types on the build be changed to `JaegerTracer.SpanBuilder`?